drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -416,6 +416,26 @@ def form_definitions(filters:)
       scope.order(updated_at: :desc)
     end
 
```

Comment:
Can we add a `deprecation_reason` to the `form_definitions` query, since it is not used anymore?